reporting/README.md

```diff
@@ -0,0 +1,43 @@
+# Test Result Reporting
+
+## Environment Setup
+
+To build and run the test reporting, first install and run either [miniconda](https://docs.conda.io/en/latest/miniconda.html) or full [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html)
+
+
+Once that is installed you can create the reporting environment with the following command:
+
+```bash
+$ conda env create -f environment.yml 
+```
+
+
+And then activate it:
+
+```bash
+$ conda activate reporting
+```
+
+## Run reporting
+
+The reporter can be run with the following command:
+
+```bash
+$ ./reporter.py
+```
+
+It will automatically fetch the latest interop test results, process them, and launch a server hosing the dashboard at [http://localhost:8050/](http://localhost:8050/)
```

Comment:
```suggestion
It will automatically fetch the latest interop test results, process them, and launch a server hosting the dashboard at [http://localhost:8050/](http://localhost:8050/)
```